setup.py

```diff
@@ -180,8 +180,8 @@ def build_man(build_cmd):
                 filename = False
 
             lang = man_dir[8:]
-            src = build_cmd.build_base  + '/data/man/' + lang  + '/gramps.1.gz'
-            target = 'share/man/' + lang + '/man1'
+            src = build_cmd.build_base  + '/data/man' + lang  + '/gramps.1.gz'
+            target = 'share/man' + lang + '/man1'
```

Comment:
Yes they have been built on windows by me and by the looks of it others.

https://github.com/gramps-project/gramps/commit/fdb8704bac98808f9f6249f4f11a39f3b998012c